### ISSUE TITLE
Blocks

### ISSUE THREAD

**Issue by chraibi** [matze/mtheme#38]
Like your theme.  I have a slight problem with the blocks. Why they are showing now "visible" color in the header? (In the demo.pdf)

**Comment by matze** [comment]
Could you elaborate a bit more? Neither the provided demo not a recompiled one has any colors. This is a recent feature that must be enabled explicitly with the `blockbg` option.

**Comment by chraibi** [comment]
how can I set the option `blockbg`?

**Comment by jdahm** [comment]
Hi chraibi,

I added this feature recently quite quickly and I didn't fully test it. Thanks for the input. I don't have a lot of time right now to work on this, but the relevant section is lines 60-70 of beamercolorthememetropolis.sty. From a quick scan it looks like I forgot to specify bg= in a few lines
```tex
\setbeamercolor{block title alerted}
\setbeamercolor{block title example}
```
and
```tex
\setbeamercolor{block title alerted}
\setbeamercolor{block title example}
```
Adding a pull request now. Please test this.

**Comment by matze** [comment]
Fixed with #41.
